src/Microsoft.Azure.WebJobs.Host/Listeners/FunctionListener.cs

```diff
@@ -138,7 +138,11 @@ public async Task StopAsync(CancellationToken cancellationToken)
                 {
                     if (_started)
                     {
+                        _logger?.LogInformation($"Stopping the listener {_listener.GetType().FullName} for function '{_descriptor.LogName}'");
```

Comment:
Let's at least cache the type name here. These calls tend to have a measurable impact.